admin-tools/src/webapp/vm/user/chef_users_confirm_remove.vm

```diff
@@ -8,8 +8,8 @@
 	<h3>	$tlang.getString("useact.remuse")
 </h3>	
 
-		#if ($alertMessage)<div class="alertMessage">$tlang.getString("useconrem.alert") $validator.escapeHtml($alertMessage)</div>#end
-		<div class="alertMessage">
+		#if ($alertMessage)<div class="sak-banner-error">$tlang.getString("useconrem.alert") $validator.escapeHtml($alertMessage)</div>#end
+		<div class="sak-banner-error">
```

Comment:
Is this an error or a warning?

Reply:
I suggest leaving this banner as a red error, based on what I have seen the alertmessage is used when there is an error in the Java functions